pandas/_typing.py

```diff
@@ -37,10 +37,7 @@
 # and use a string literal forward reference to it in subsequent types
 # https://mypy.readthedocs.io/en/latest/common_issues.html#import-cycles
 if TYPE_CHECKING:
-    from typing import (
-        TypedDict,
-        final,
-    )
+    from typing import TypedDict
```

Comment:
Could move `TypedDict` out of the `TYPE_CHECKING` block. `TypedDict` was introduced in python 3.8.

Reply:
yep. already done Literal, this PR is final, a subsequent PR will be TypedDict (not yet opened PR to avoid merge conflicts)